apps/dashboard/features/resilience-stages/components/StageContent.tsx

```diff
@@ -43,17 +52,28 @@ export const StageContent = ({
               Requirements
```

Comment:
Potential runtime error: The code accesses `daoConfigByDaoId[selectedDaoId]` without checking if the selectedDaoId exists in the config object. If an invalid daoId is passed in the URL params, this will throw a runtime error when trying to access `.name` property of undefined.